composer.json

```
@@ -69,6 +70,10 @@
         {
             "type": "composer",
             "url": "https://packages.drupal.org/8"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/openeuropa/task-runner-ensure-behat-batching"
```

You can add 
```suggestion
            "url": "https://github.com/openeuropa/task-runner-ensure-behat-batching",
            "no-api": true
```
as a temporary solution to avoid failures on the build.